dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/CamelCommand.java

```diff
@@ -67,9 +68,28 @@ public Integer call() throws Exception {
             configureLoggingOff();
         }
 
+        replacePlaceholders();
+
         return doCall();
     }
 
+    private void replacePlaceholders() throws Exception {
+        for (CommandLine.Model.ArgSpec argSpec : spec.args()) {
+            String defaultValue = spec.defaultValueProvider().defaultValue(argSpec);
```

Comment:
the default values are the values set in configuration? (like when set with `jbang camel@apache/camel config set repos=https://maven.repository.redhat.com/ga`)

Reply:
exactly, the `defaultValueProvider()` loads the value from the properties file (updated via `camel config set`)